.changeset/late-suits-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@rocket.chat/meteor': patch
+---
+
+Fixed issue where recentList was not always an array, causing a TypeError on emoji recent list updates
```

Comment:
As per [our docs](https://developer.rocket.chat/docs/development-workflow)

> A changeset is information about changes made in a branch or commit that will be present in the release's changelog.

We must avoid jargon when writing changesets. In my own words, they must be as simple as possible without ommiting information.